BridgeCareApp/BridgeCare/Services/Simulations.cs

```diff
@@ -97,14 +95,35 @@ public int CreateNewSimulation(int networkId, string simulationName, BridgeCareC
                     WEIGHTING = "none",
                     COMMITTED_START = DateTime.Now.Year,
                     COMMITTED_PERIOD = 5,
-                    DATE_LAST_RUN = DateTime.Now
+                    TREATMENTS = new List<TREATMENT>
+                    {
+                        new TREATMENT()
+                        {
+                            TREATMENT1 = "No Treatment",
+                            BEFOREANY = 1,
+                            BEFORESAME = 1,
+                            BUDGET = null,
+                            DESCRIPTION = "Default Treatment",
+                            OMS_IS_EXCLUSIVE = null,
+                            OMS_IS_REPEAT = null,
+                            OMS_REPEAT_START = null,
+                            OMS_REPEAT_INTERVAL = null,
+                            CONSEQUENCES = new List<CONSEQUENCE>
+                            {
+                                new CONSEQUENCE
+                                    {
+                                    ATTRIBUTE_ = "AGE",
+                                    CHANGE_ = "+1"
```

Comment:
Same comment as earlier for strings "AGE", "+1".

Reply:
will make a card

Reply:
ok